rpm/poppler.spec

```diff
@@ -156,3 +158,7 @@ rm -f %{buildroot}%{_libdir}/*.la
 %defattr(-,root,root,-)
 %{_bindir}/pdf*
 %{_mandir}/man1/*
+
+%exclude
+%{_datarootdir}/locale/ca/LC_MESSAGES/pdfsig.mo
```

Comment:
Commonly these are in one line like "%exclude /foo/bar". 

Interesting thing this one separate catalan translation in the whole repository.

Reply:
The maintainers are really in to their translations, and probably Catalan.

Reply:
onelined as well